docs/rat.html

```diff
@@ -643,6 +643,10 @@ <h2 id="operator-precedence-table">Operator Precedence Table</h2>
   <td><code>+</code>, <code>-</code></td>
   <td style="text-align:center;">10</td>
 </tr>
+<tr>
+  <td><code>&gt;</code>, <code>&lt;</code>, <code>&gt;=</code>, <code>&lt;=</code>, <code>!=</code>, <code>==</code></td>
+  <td style="text-align:center;">5</td>
+</tr>
```

Comment:
Eh, I don't find them particularly problematic since I have to pull after pushing for formatting changes anyway. But if you want to change how they work I guess it's better than keeping these chonky html, javascript files on every PR.